tests/Rector/MethodCall/GetToConstructorInjectionRector/Fixture/fixure3.php.inc

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\MethodCall\GetToConstructorInjectionRector\Fixture;
+
+use Symfony\Bundle\FrameworkBundle\Controller\BaseController;
+
+class ClassWithNamedService extends BaseController
+{
+    public function render()
+    {
+        $someService = $this->get('some_service');
+
+        $this->renderTwig([
+            'posts' => $this->get('some_service')->callMe()
+        ]);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\Rector\MethodCall\GetToConstructorInjectionRector\Fixture;
+
+use Symfony\Bundle\FrameworkBundle\Controller\BaseController;
+
+class ClassWithNamedService extends BaseController
+{
+    public function __construct(private \stdClass $stdClass, private \stdClass $formFactory)
```

Comment:
The form factory cannot be here, as it's handled by the Symfony controller itself:

```suggestion
    public function __construct(private \stdClass $stdClass)
```